docs/source/onnxruntime/usage_guides/amdgpu.mdx

```diff
@@ -0,0 +1,117 @@
+# Accelerated inference on AMD GPUs
+
+By default, ONNX Runtime runs inference on CPU devices. However, it is possible to place supported operations on an AMD Instinct GPU, while leaving any unsupported ones on CPU. In most cases, this allows costly operations to be placed on GPU and significantly accelerate inference.
+
+This guide will show you how to run inference on the `ROCMExecutionProvider` execution provider that ONNX Runtime supports for AMD GPUs:
+
+## Installation
+
+#### 1. ROCM Installation (V 5.7.X)
```

Comment:
```suggestion
#### 1. ROCm Installation (V 5.7.X)
```
`(V 5.7.X)` means that ROCm 5.7 is a requirement? Can we put that rather in a sentence?

Reply:
The branch AMD team shared for onnxruntime was on ROCm 5.7. So this has been tested on the ROCm5.7. I could mention that the following instructions are torun the ROCMEP on AMD gpu with ROCm 5.7 installed.